appengine/Rakefile

```diff
@@ -0,0 +1,19 @@
+$VERBOSE = true
```

Comment:
Instead of setting this globally, maybe you can use rakes `verbose` helper in a specific place - what's this for?